docs: Added Status to Transactions Concept Page (#112)
publicsq/developers.publicsquare.com: docs/concepts/transactions.mdx

| `pending`          | Used for ACH payments and payouts. Once the funds have cleared the bank account, the status will be updated to `successful` usually 2-3 days after the transaction was processed. |
| `rejected`         | Set when the `fraud_details.decision` field on a payment is marked as `reject`. This is set when your fraud decision rules trigger a `reject` decision.                           |
| `declined`         | Set when the bank rejects the payment or payout. Usually due to AVS or CVV2 checks failing or the payment method being declined such as invalid card or bank account details.     |
| `cancelled`        | Set when a payment, payout, or refund has been successfully cancelled before being settled. 

> | `error`            | The payment or refund attempt failed due to an unexpected system or integration issue, such as an invalid response from the processor, network errors, or a missing required parameter. This status applies to all payment types.|

Reviewer: The table formatting needs fixed. The trailing pipe was lost on the `cancelled` row and all of the table column alignments need fixed. Use `Markdown Table Formatter` extension in VSCode to fix this.

> | `error`            | The payment or refund attempt failed due to an unexpected system or integration issue, such as an invalid response from the processor, network errors, or a missing required parameter. This status applies to all payment types.|

Reviewer: ```suggestion
| `error`            | The payment, payout, or refund attempt failed due to an unexpected system or integration issue, such as an invalid response from the processor, network errors, or a missing required parameter. |
```
